web/src/views/Game/Team.tsx

```diff
@@ -39,21 +39,14 @@ export default function Team(props: TeamProps) {
           <Tab label="Team Stats" />
           <Tab label="Graphs" />
         </Tabs>
-        { tab === 0 &&
+        {tab === 0 && (
           <Paper>
             <Roster players={players} />
           </Paper>
```

Comment:
should Paper get pulled down to make this consistent with the others? why isn't it?